app/authorization/server/startup.js

```diff
@@ -26,6 +26,7 @@ Meteor.startup(function() {
 		{ _id: 'change-livechat-room-visitor',       roles: ['admin', 'livechat-manager', 'livechat-agent'] },
 		{ _id: 'create-c',                           roles: ['admin', 'user', 'bot', 'app'] },
 		{ _id: 'create-d',                           roles: ['admin', 'user', 'bot', 'app'] },
+		{ _id: 'create-not-cl',                      roles: ['admin', 'owner', 'leader'] },
```

Comment:
I had a look in the code of this file. 
But, I did not find a role called `leader`.
Can you tell me which role is it?